tensorflow/src/main/scala/magnolify/tensorflow/ExampleType.scala

```diff
@@ -55,12 +55,13 @@ object ExampleType {
 sealed trait ExampleField[T] extends Serializable {
   @transient private lazy val schemaCache: concurrent.Map[ju.UUID, Schema] =
     concurrent.TrieMap.empty
+
+  protected def buildSchema(cm: CaseMapper): Schema
+  def schema(cm: CaseMapper): Schema =
+    schemaCache.getOrElseUpdate(cm.uuid, buildSchema(cm))
+
   def get(f: Features, k: String)(cm: CaseMapper): Value[T]
   def put(f: Features.Builder, k: String, v: T)(cm: CaseMapper): Features.Builder
-
-  def schema(k: String)(cm: CaseMapper): Schema =
-    schemaCache.getOrElseUpdate(cm.uuid, buildSchema(k)(cm))
```

Comment:
This is not correct: we leak the upper key into the schema. For primitives for instance, schema will be cached with the name of the 1st fied of this type.
Schema should not contain name information of enclosing class